server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -19,106 +20,102 @@ import {
-        if(result?.data) {
-            isolationSourcesLogger.info('got response from DB', Severity.LOW);
-            response.send(result?.data?.allIsolationSources?.nodes)
-        } else {
-            isolationSourcesLogger.error(`couldnt query isolation sources due to ${result?.errors[0]?.message}`, Severity.HIGH);
-            response.sendStatus(errorStatusCode);
-        }
+
+    isolationSourcesLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_ISOLATION_SOURCES, response.locals)
+    .then(result => {
+        isolationSourcesLogger.info(validDBResponseLog, Severity.LOW);
+        response.send(result?.data?.allIsolationSources?.nodes || []);
     }).catch(error => {
-        isolationSourcesLogger.error(`got error when approaching the graphql API: ${error}`, Severity.HIGH);
-        response.sendStatus(errorStatusCode);
+        isolationSourcesLogger.error(invalidDBResponseLog(error), Severity.HIGH);
+        response.sendStatus(error);
     });
 });
 
 const convertClinicalDetailsFromDB = (result: GetInvestigatedPatientClinicalDetailsFields) => {
     const clinicalDetails = result.data.investigationByEpidemiologyNumber;
     const convertedClincalDetails = {
         ...clinicalDetails,
-        isolationAddress: clinicalDetails.addressByIsolationAddress,
-        symptoms: clinicalDetails.investigatedPatientSymptomsByInvestigationId.nodes
+        symptoms: clinicalDetails.symptoms.nodes
             .map((symptom: any) => symptom.symptomName),
         ...clinicalDetails.investigatedPatientByInvestigatedPatientId,
         backgroundDiseases: clinicalDetails.investigatedPatientByInvestigatedPatientId.investigatedPatientBackgroundDiseasesByInvestigatedPatientId.nodes
             .map((backgroudDisease: any) => backgroudDisease.backgroundDeseasName)
     }
 
-    delete convertedClincalDetails.addressByIsolationAddress;
-    delete convertedClincalDetails.investigatedPatientSymptomsByInvestigationId;
     delete convertedClincalDetails.investigatedPatientByInvestigatedPatientId;
     delete convertedClincalDetails.investigatedPatientBackgroundDiseasesByInvestigatedPatientId;
 
     return convertedClincalDetails;
 }
 
 clinicalDetailsRoute.get('/getInvestigatedPatientClinicalDetailsFields', (request: Request, response: Response) => {
+    
     const fetchClinicalDetailsFieldsLogger = logger.setup({
-        workflow: 'Fetching Clinical Details',
+        workflow: 'qurey investigation clinical details',
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     })
-    fetchClinicalDetailsFieldsLogger.info('launching DB request', Severity.LOW);
-    graphqlRequest(GET_INVESTIGATED_PATIENT_CLINICAL_DETAILS_BY_EPIDEMIOLOGY_NUMBER, response.locals, { epidemiologyNumber: +request.query.epidemiologyNumber }).then(
+
+    const parameters = { epidemiologyNumber: +request.query.epidemiologyNumber };
```

Comment:
can you use parseInt instead of unary +? 
I know it's pretty esoteric but take [this ](https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which) example from stackoverflow :
> +"" -> 0
> parseInt("") -> NaN

Reply:
this goes for alot of files in here